tests/test_create_flatfield.py

```diff
@@ -93,11 +93,15 @@ def test_create_flatfield_neptune():
     
     print(flatdivided_dataset[0].ext_hdr)
 
+    corgidrp.track_individual_errors = old_err_tracking
+
     return
     
      #creating flatfield using uranus for band4 
     
 def test_create_flatfield_uranus():
+
+    corgidrp.track_individual_errors = True
```

Comment:
The unit tests were written a long time ago using this. There are other step function unit tests that were written that require track_individual_errors as True, before we set False as the default. I merely explicitly declared it here, because it  was implicitly being set to True by the previous unit test (test_create_flatfield_neptune), and I didn't want this unit test to fail in the future, if the auto-ordering of pytest unit tests changes. 

The e2e test is done with track_individual_errors = False, as we prefer.